src/dbcp/metadata/private_data_warehouse.py

```diff
@@ -14,6 +15,86 @@
+    Column("phase_name", String),
+    Column("resource", String),
+    Column("developers", String),
+    Column("owners", String),
+    Column("iso_region", String),
+    Column("owner_types", String),
+    Column("capacity_mw", Float),
+    Column("state_id_fips", String),
+    Column("county_id_fips", String),
+    Column("avg_latitude", Float),
+    Column("avg_longitude", Float),
+    Column("geocoded_locality_name", String),
+    Column("geocoded_locality_type", String),
+    Column("geocoded_containing_county", String),
+    Column("geocoded_county_id_fips", String),
+    Column("geocoded_state_id_fips", String),
+    Column("census_county_id_fips", String),
+    Column("raw_mw_total_capacity", Float),
+    Column("raw_units_total", String),
+    Column("raw_mw_online_capacity", String),
+    Column("raw_mw_under_construction_capacity", Float),
+    Column("raw_mw_adv_development_capacity", Float),
+    Column("raw_mw_decommissioned_capacity", Float),
+    Column("raw_units_online", Integer),
+    Column("raw_units_under_construction", Integer),
+    Column("raw_units_adv_development", Integer),
+    Column("raw_units_decommissioned", Integer),
+    Column("raw_years_online", String),
+    Column("raw_quarters_online", String),
+    Column("raw_countries", String),
+    Column("raw_states", String),
+    Column("raw_counties", String),
+    Column("raw_community", Boolean),
+    Column("raw_repower_type", String),
+    Column("raw_repower_phases", String),
+    Column("raw_owner_types", String),
+    Column("raw_constructors", String),
+    Column("raw_servicers", String),
+    Column("raw_manufacturers", String),
+    Column("raw_models", String),
+    Column("raw_hub_heights", String),
+    Column("raw_rotor_diameters", String),
+    Column("raw_total_heights", String),
+    Column("raw_avg_latitude", String),
+    Column("raw_avg_longitude", String),
+    Column("raw_offtake_types", String),
+    Column("raw_power_purchasers", String),
+    Column("raw_power_purchaser_types", String),
+    Column("raw_power_purchaser_type_details", String),
+    Column("raw_ppa_rates", String),
+    Column("raw_ppa_start_years", String),
+    Column("raw_ppa_end_years", String),
+    Column("raw_ppa_durations", String),
+    Column("raw_interconnected_utilities", String),
+    Column("raw_iso_rtos", String),
+    Column("raw_nercs", String),
+    Column("raw_balancing_authorities", String),
+    Column("raw_state_houses", String),
+    Column("raw_state_senates", String),
+    Column("raw_congressional_districts", String),
+    Column("raw_secondary_offtake_types", String),
+    Column("raw_secondary_purchaser_ids", String),
+    Column("raw_secondary_purchaser_types", String),
+    Column("raw_tracking_types", String),
+    Column("raw_ilrs", Float),
+    Column("raw_nameplate_mws", String),
+    Column("raw_storage_energies", String),
+    Column("raw_storage_durations", String),
+    schema=schema,
+)
+
 acp_projects = Table(
     "acp_projects",
```

Comment:
So this table really should have a name that indicates what type of "time" it contains, meaning is it changelog, snapshots, current state or timeseries? I believe this is current state

Reply:
This is the current state table. Previously, it was the only table that we tracked with ACP data.